FrontEnd/src/App.css

```diff
@@ -1,5 +1,5 @@
 .App {
-  text-align: center;
+  /*text-align: center; This style breaks all text on the main page. I propose to remove this*/
```

Comment:
If it doesn't correspond to the design it's better to remove it, also please remove commented code bellow